docs/sources/datasources/mysql.md

```diff
@@ -200,20 +201,46 @@ GROUP BY time
 ORDER BY time
 ```
 
+Data frame result:
```

Comment:
@torkelo any input?

Reply:
I created a section on this at https://grafana.com/docs/grafana/latest/basics/timeseries-dimensions/#multiple-dimensions-in-table-format , the idea was that it could be a shared explanation that the SQL data sources could link to for the conceptual parts.